src/permission-groups/permission-groups.service.spec.ts

```diff
@@ -72,6 +72,42 @@ describe('PermissionGroupsService', () => {
     expect(service).toBeDefined();
   });
 
+  describe('findOne', () => {
+    let findAssetSpy: jest.SpyInstance;
+    let mockAsset: MockAsset;
```

Comment:
```suggestion
    let mockAsset: Mocked<Asset>;
    
    beforeEach(() => {
      mockAsset = createMock<Asset>()
```
Do you try auto mock? I think they work better with TS and are easier to maintain then our manual mocks.